Code/Engine/RendererVulkan/Device/Implementation/DeviceVulkan.cpp

```diff
@@ -307,21 +467,109 @@ ezResult ezGALDeviceVulkan::ShutdownPlatform()
   }
 
   m_pDefaultPass = nullptr;
+
+  ezMemoryAllocatorVulkan::DeInitialize();
```

Comment:
Since we have this allocator thingy, can it report live resources (buffers, images) on deinit?

Reply:
Allocator and validation layer will do that for you automatically.